src/lib/stores/passage-form.store.ts

```diff
@@ -1,7 +1,7 @@
 import { writable } from 'svelte/store';
-import type { FrontendPassagesByBook } from '$lib/types/passage';
+import type { BibleSection, FrontendPassagesByBook } from '$lib/types/passage';
 
-export const selectedId = writable<string>('default');
+export const selectedBibleSection = writable<BibleSection | null>(null);
```

Comment:
Instead of tracking the selected passage id, we now need to track the selected Bible section as a whole. This will let things work with both the CBBT-ER passage-based approach as well as the more freeform Bible section selection approach.